### ISSUE TITLE
train_ssd.py loss=NaN or The program does not continue or stop

### ISSUE THREAD

**Issue by chenaifang** [HiKapok/SSD.TensorFlow#119]
When I am training my own data, occasionally loss=nan occurs. Your two solutions for loss=nan are useless;In other experiments where loss=nan does not occur, the program will not run nor stop.The best training result was 0.091, and it happened more than once.

**Some parameters that have been modified:**
![11](https://user-images.githubusercontent.com/34065949/63143820-b9caaf80-c022-11e9-98c2-601ccfee0397.png)

**The program didn't work, and pressing Ctrl+C couldn't stop it, so I had to disconnect the server to end the program**

![TLC@SH$ VXVIYZI3C2DB`@W](https://user-images.githubusercontent.com/34065949/63143840-d49d2400-c022-11e9-93d2-5e38f51899c3.png)

**The result of running _python3 eval ssd.py_**
![N7W LSCX7` CS1EI4 Z{JO4](https://user-images.githubusercontent.com/34065949/63143968-66a52c80-c023-11e9-8468-3fc0ffbd2bfe.png)

**Results of running _python3 voc eval. py_**

![07P$D JTVF9L WKS(M3SV3Y](https://user-images.githubusercontent.com/34065949/63143905-204fcd80-c023-11e9-9ead-815c23e321e9.png)

**Comment by HiKapok** [comment]
@chenaifang are you using voc or yourself dataset?

**Comment by chenaifang** [comment]
I'm using my own data set.   @HiKapok

**Comment by HiKapok** [comment]
@chenaifang as earlier issues mentioned, you must ensure every image should have at least **one** valid gt. And your loss value is not correct, you should check carefully. If you finnally get normal loss value(~20), you can further lower learning rate if NAN still exists.
